src/api/delete-user.js

```diff
@@ -0,0 +1,28 @@
+async function deleteUser(username) {
+    // const url = `${import.meta.env.VITE_API_URL}/users/${username}`;
```

Comment:
pointing at local (just reminded me to change in my upcoming PR!)

Reply:
Fixed :)